transform/mattermost-analytics/models/staging/salesforce/stg_salesforce__account.sql

```diff
@@ -0,0 +1,179 @@
+
+with source as (
```

Comment:
This models should be added to `_salesforce__models.yml`.

Reply:
Added account to `_salesforce__models.yml` file